region/maridia/outer.json

```diff
@@ -1994,6 +2102,84 @@
                   "devNote": "The second jump is actually doable with just a regular SpringBall jump but is ridiculously tight."
+                },
+                {
+                  "name": "Suitless Water Shinecharge Vertical Spark",
+                  "notable": false,
+                  "requires": [
+                    "canWaterShineCharge",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 43,
+                        "excessShinesparkFrames": 9,
+                        "openEnd": 2
+                      }},
+                      {"and": [
+                        "canMidairShinespark",
+                        {"canShineCharge": {
+                          "usedTiles": 33,
+                          "shinesparkFrames": 38,
+                          "excessShinesparkFrames": 9,
+                          "openEnd": 2
+                        }}
+                      ]}
+                    ]},
+                    {"adjacentRunway": {
+                      "fromNode": 3,
+                      "usedTiles": 0.5,
+                      "physics": ["normal"],
+                      "overrideRunwayRequirements": true,
+                      "useFrames": 80
+                    }}
+                  ],
+                  "note": [
+                    "Start the dash as close to the door transition as possible in order to have a larger window for storing the shinecharge.",
+                    "Spark up and fall onto the platform left of the fish."
+                  ]
+                },
+                {
+                  "name": "Suitless Water Shinecharge Diagonal Spark",
+                  "notable": false,
+                  "requires": [
+                    "canWaterShineCharge",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 43,
+                        "excessShinesparkFrames": 26,
+                        "openEnd": 2
+                      }},
+                      {"and": [
+                        "canMidairShinespark",
+                        {"canShineCharge": {
+                          "usedTiles": 33,
+                          "shinesparkFrames": 38,
+                          "excessShinesparkFrames": 25,
+                          "openEnd": 2
+                        }}
+                      ]}
+                    ]},
+                    {"adjacentRunway": {
+                      "fromNode": 3,
+                      "usedTiles": 0.5,
+                      "physics": ["normal"],
+                      "overrideRunwayRequirements": true,
+                      "useFrames": 80
+                    }},
+                    {"or": [
+                      "canCarefulJump",
```

Comment:
Getting around the pirate is some advanced tech, a little more than a tricky jump, with crouch jump.  Or tricky+morph.

Reply:
I now see why this is so difficult. There are only a handful of useful strats leaving 6, and they mostly all make this not important:
midair spring ball up left,
going up right and then using a midair spring ball or the insane jump or plasma+ice ...

The only strat that doesnt play nice is using the frozen fish to get up left. The reason this one is annoying is because it is far easier to get over the pirate if the fish is already killed.